model/src/main/proto/interaction_object.proto

```diff
@@ -53,5 +54,30 @@ message Fraction {
 
 // Structure for a ListOfString object.
 message ListOfSetsOfHtmlStrings {
-  repeated StringList set_of_html_strings = 1;
+  StringList set_of_html_strings = 1;
+}
+
+// Structure for a ImageWithRegions object.
+message ImageWithRegions {
```

Comment:
Here & elsewhere: shouldn't this just be ``ImageWithRegion``? It represents a single instance from what I can tell.

Reply:
it can be multiple regions